tests/unit/RestfulServerTest.php

```diff
@@ -53,6 +53,10 @@ protected function setUp()
     {
         parent::setUp();
         Director::config()->set('alternate_base_url', $this->baseURI);
+        RestfulServer::config()->set('authenticators', [
+            BasicRestfulAuthenticator::class,
+            EmailOnlyAuthenticator::class,
```

Comment:
We shouldn't add this authenticator to *all* tests if they don't need/want it as it may result in unintended side-affects in future tests. Just add it to the specific test for this situation.